lessons/specifics/plug.md

```diff
@@ -51,7 +53,101 @@ defmodule HelloWorldPlug do
 end
 ```
 
-## Creating a Plug
+Save the file to `lib/example/hello_world_plug.ex`.
+
+Since we're starting a Plug application from scratch, we need to define the application module.
+Update `lib/example.ex` to start and supervise Cowboy:
+
+```elixir
+defmodule Example do
+  use Application
+  require Logger
+
+  def start(_type, _args) do
+    children = [
+      Plug.Adapters.Cowboy.child_spec(:http, Example.HelloWorldPlug, [], port: 8080)
+    ]
+
+    Logger.info "Started application"
+
+    Supervisor.start_link(children, strategy: :one_for_one)
+  end
+end
+```
+
+This supervises Cowboy, and in turn, supervises our `HelloWorldPlug`.
+
+Now, the `application` portion of `mix.exs` needs two things: 1) A list of
```

Comment:
I'm not sure if its official in the styleguide but I believe we try to do one sentence per line unless its a unique case. CC @doomspork for this because I cant remember off the top of my head, and its really late and it might be me mixing that up with my own markdown writing style.